src/content/docs/fr/reference/dev-toolbar-app-reference.mdx

```diff
@@ -83,6 +83,30 @@ Chemin d'accès au fichier qui exporte l'application de la barre d'outils de dé
 }
 ```
 
+<p><Since v="5.0.0" /></p>
+
+La fonction accepte également une `URL` comme point d'entrée (`entrypoint`) :
+
+
+```ts title="my-integration.js" "new URL("./my-app.js", import.meta.url)"
```

Comment:
I translated the string in the example but I forgot to update the highlighting...
````suggestion
```ts title="mon-integration.js" "new URL("./mon-appli.js", import.meta.url)"
````